tests/sentry/utils/sdk_crashes/test_sdk_crash_detection.py

```diff
@@ -19,6 +21,14 @@
 from sentry.utils.sdk_crashes.sdk_crash_detection import sdk_crash_detection
 
 
+@pytest.fixture
+def store_event(default_project, factories):
```

Comment:
Is there any reason preventing us from calling the function regularly, and making it a fixture instead?

Reply:
The function needs the `default_project` and the `factories` fixture. I turned it `store_event` into a fixture cause then I only need to pass the data as an argument. I'm still new to Python fixtures. Please let me know if you have suggestions on how to improve this, @iker-barriocanal.

Reply:
I don't know much about fixtures either, I was just curious on this 😄